src/Dialogs/Airspace/dlgAirspaceWarnings.cpp

```diff
@@ -330,6 +330,7 @@ AirspaceWarningListWidget::OnPaintItem(Canvas &canvas,
   auto [text_rc, altitude_rc] =
     text_altitude_rc.VerticalSplit(text_altitude_rc.right - (padding + altitude_width));
   text_rc.right -= padding;
+  altitude_rc.bottom = altitude_rc.bottom;  // unused variable
```

Comment:
This isn't an unused variable. `altitude_rc` is a variable, but `altitude_rc.bottom` is not.

Reply:
My compiler says that altitude_rc is an unused variable, and so i made this fix ... :-( 
Or should I only change the comment?

Reply:
See, it's about `altitude_rc`, not about `altitude_rc.bottom`.
If the variable is unused, remove it instead of applying a workaround.

Reply:
Done